site.yml

```diff
@@ -72,6 +72,7 @@
         - ansible_post_tasks is defined
         - ansible_post_tasks is not none
         - ansible_system is match("Linux")
+        - ansible_post_tasks is match("^(http|https)://.*")
```

Comment:
Should we include `file://` too?

Reply:
I think if it's a local file, we also do not want it to run download tasks.

Reply:
I think we will if it's file:// it'll move it to the right name